hawkular-inventory-parent/hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/InventoryService.java

```diff
@@ -37,4 +37,6 @@
     Collection<Resource> getResourcesByType(String typeId);
     Optional<ResourceType> getResourceType(String typeId);
     Optional<String> getAgentConfig(String resourceType);
```

Comment:
the agent isn't giving a "resource type" when it asks for its config - its giving some kind of "server identification" - like "EAP 7.0.0.Final" or "Fuse 1.0.1.CP2" or "My Custom App".